integrations/pjrt/src/iree_pjrt/common/command_line_utils.cc

```diff
@@ -0,0 +1,54 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "command_line_utils.h"
+
+namespace iree {
+namespace pjrt {
+
+// TODO: currently this function doesn't handle escape sequences,
+// it just ensure that single/double quotes are interpreted corrently.
+std::optional<std::vector<std::string>> ParseOptionsFromCommandLine(
+    std::string_view options_str) {
```

Comment:
Ahh thank you for your information. I'll check these code in iree::base.

Reply:
Hmm I've checked related code but it seems these functions in iree:base need some changes before we can reuse here? Not sure if missed something.

Reply:
It's fine if that code can't be used here. Just 50 lines of new code here... but flag parsing and string manipulation tends to have lots of edge cases and it can be helpful to have a single well tested implementation.